airflow/jobs.py

```diff
@@ -1774,8 +1774,11 @@ def _execute(self):
 
         # create dag runs
         dr_start_date = start_date or min([t.start_date for t in self.dag.tasks])
-        next_run_date = self.dag.normalize_schedule(dr_start_date)
         end_date = end_date or datetime.now()
+        # next run date for a subdag isn't relevant (schedule_interval for subdags
+        # is ignored) so we use the dag run's start date in the case of a subdag
+        next_run_date = self.dag.normalize_schedule(dr_start_date) \
+            if "." not in self.dag.dag_id else dr_start_date
```

Comment:
Have a look at:

https://github.com/apache/incubator-airflow/blob/master/airflow/models.py#L201

I think it is actually a bug that it is not properly exposed. Can you please add it to the "DAG class", I think that to be a more proper fix.